orchagent/mirrororch.h

```diff
@@ -93,7 +93,13 @@ class MirrorOrch : public Orch, public Observer, public Subject
     bool updateSessionDstMac(const string&, MirrorEntry&);
     bool updateSessionDstPort(const string&, MirrorEntry&);
     bool updateSessionType(const string&, MirrorEntry&);
-    bool setSessionState(const string&, MirrorEntry&);
+    void setSessionStatus(const string&, MirrorEntry&);
```

Comment:
I think only the `setSessionStatus` function is needed while it will store all the related fields.